### PR TITLE
added astr 128 requirements

### PR DESCRIPTION
**Pull request by kareemelbadry**
I added a few requirements, all available via pip from PyPI, for the Astr 128/256 class in spring 2019.

### CODE REVIEW

**Comment by yuvipanda** [comment]
@kareemelbadry sure! First you need to get a copy of this repo (rather than your local fork) on your machine. You can do that with:

`git remote add official https://github.com/berkeley-dsep-infra/datahub.git`

This adds a new 'remote' reference called 'official' pointing to this repo. There is already a default one called 'origin' pointing to your fork. You can see this with `git remote -v`

You can fetch the current contents of this repo with `git fetch official`.

Then, you can rebase on top of staging from here with: `git rebase official/staging`. 

Once that is done, you can force push to your branch on your fork. Force pushing should be carefully used only on forks. `git push -f origin <your-branch-name>`.

That should update this PR.

Let me know if that works

**Comment by yuvipanda** [comment]
Awesome, @kareemelbadry. I'm a bit swamped today, will give you the appropriate rights and instructions on monday. Does that work?

**Comment by kareemelbadry** [comment]
Thanks Yuvi! That sounds great.

Kareem

File: deployments/datahub/image/requirements.txt [hunk 1]
```diff
@@ -98,3 +98,10 @@ git+git://github.com/statlab/cryptorandom.git@0721472
 plotly==3.4.0
 cufflinks==0.14.6
 
+# astr 128/256; spring 2019
+scipy==0.19.1
+astroquery==0.3.9
+astropy==3.1.1
+dustmaps==1.0.3
+pystan==2.18.1.0
+
```
Comment (profjsb) [inline deployments/datahub/image/requirements.txt]: perhaps `corner` as well?

Comment (kareemelbadry) [reply deployments/datahub/image/requirements.txt]: corner is alread included in requirements.txt, presumably added for another class.

**Comment by yuvipanda** [comment]
@kareemelbadry I've added you as a collaborator to this repo. You should:

1. Merge this PR, and watch https://circleci.com/gh/berkeley-dsep-infra/datahub/tree/staging to see progress on deployment to staging.datahub.berkeley.edu. If the deployment succeeds, go to staging.datahub.berkeley.edu, login and test that the libraries you installed work as they should. If you already had a running notebook there, you might have to go to 'control panel -> stop server' followed by 'start server' to get the updated image.

2. If staging works alright, you can make a new PR to merge to production using this link: https://github.com/berkeley-dsep-infra/datahub/compare/prod...staging. Make sure only your commits are present there. Make the PR, and merge it. Watch https://circleci.com/gh/berkeley-dsep-infra/datahub/tree/prod to see it deploy, and once it succeeds, go to datahub.berkeley.edu to go check it out! 

If you run into issues, go to uctech.slack.com and ping me or ryan in the #jupyter channel there.

Note also that we're doing pre-semester maintenance on the cluster on Jan 15 and 16, so it might be wonky. If you are doing this during one of those two days, please check on slack first to get a go-ahead before doing this.

**Comment by yuvipanda** [comment]
We're doing maintenance today and tomorrow, so hold off on merging until we give you a go-ahead.

**Comment by yuvipanda** [comment]
@profjsb as it is set up, all people courses on datahub.berkeley.edu will have access to these libraries. We can also et up an astrolab.datahub.berkeley.edu if you'd prefer that. Having it all in the same hub is the easiest thing to do right now though.

**Comment by ryanlovett** [comment]
@profjsb A separate hub is really only critical if any of the datahub admins (i.e. Data 8 GSIs or UGSIs) are students in your course, or vice versa.

**Comment by yuvipanda** [comment]
@kareemelbadry our maintenance and stuff is done, you are welcome to merge this whenever you wish.

**Comment by kareemelbadry** [comment]
Thanks Yuvi. Currently it says "Merging is blocked" and "You’re not authorized to merge this pull request." Do I need some some other authorization?

**Comment by yuvipanda** [comment]
@kareemelbadry hmm, do you still see that? I see you have been given write access to this repo.

**Comment by kareemelbadry** [comment]
> @kareemelbadry hmm, do you still see that? I see you have been given write access to this repo.

Huh, I still see it. 

When I hover over my name in this thread, I see "You have been invited to collaborate on the datahub repository." 

I'm not sure whether I have to do anything to accept the invitation. When I go to https://github.com/berkeley-dsep-infra/datahub/invitations (as suggested elsewhere on the web), it just redirects to https://github.com/berkeley-dsep-infra/datahub. 

Sorry for the confusion.

**Comment by yuvipanda** [comment]
@kareemelbadry hmm, it tells me you are a collaborator here too. Maybe try logging out of github and back on?

**Pull request merged**